pkg/controllers/job/job_controller_actions.go

```diff
@@ -152,11 +152,18 @@ func (cc *jobcontroller) killJob(jobInfo *apis.JobInfo, podRetainPhase state.Pha
 	}
 
 	// Delete PodGroup
-	pgName := job.Name + "-" + string(job.UID)
-	if err := cc.vcClient.SchedulingV1beta1().PodGroups(job.Namespace).Delete(context.TODO(), pgName, metav1.DeleteOptions{}); err != nil {
+	pgName := jobhelpers.GetRelatedPodGroupName(job)
+	pgIface := cc.vcClient.SchedulingV1beta1().PodGroups(job.Namespace)
+	if err := pgIface.Delete(context.TODO(), pgName, metav1.DeleteOptions{}); err != nil {
```

Comment:
Why named Iface here?

Reply:
I mean pgIface itself is bit ambiguous: )

Reply:
I noticed `cc.vcClient.SchedulingV1beta1().PodGroups(job.Namespace)` returns a `v1beta1.PodGroupInterface` type, so i set the variable name to pgIface(v1beta1.**P**od**G**roup**I**nter**face**), can you give me a suggestion 🙇🏼

Reply:
I think pgClient is ok.